server/src/score/score.entity.ts

```diff
@@ -0,0 +1,101 @@
+import { Application } from '@/application/application.entity';
+import { User } from '@/user/user.entity';
+import { Entity, Column, PrimaryGeneratedColumn, OneToOne, JoinColumn } from 'typeorm';
+import { CustomBaseEntity } from '../common/custom-base.entity';
+
+@Entity({
+  name: 'pbgp_score',
+})
+export class Score extends CustomBaseEntity {
```

Comment:
Should we have a base table for this? Like BroaderReviewScores or something and have all the user and mappings in this table?

Reply:
Added ScoreBaseEntity that contains all the fields but not mappings. Can be reused for WorkshopScore later.